### ISSUE TITLE
support downloading experiment data

### ISSUE THREAD

**Issue by oegeshipster** [langchain-ai/langsmith-sdk#1318]
### Feature request

Although the langsmith experiment viewing interface is pretty good for basic usecases, for my (quite complicated) usecase I would like to build a custom offline experiment comparison tool (with e.g. streamlit or dash or similar). However I do not see a method in the sdk to download a given experiment run from the platform?

### Motivation

In order to do error analysis on runs with complicated nested structured outputs from images, I need an interface that allows me to quickly compare specific metrics for each extracted field and e.g. highlight them in the source pdf, etc. I can build this myself, but need access to the underlying experiment result.

**Comment by oegeshipster** [comment]
or I guess this might be covered by this PR? https://github.com/langchain-ai/langsmith-sdk/pull/1280

**Comment by hinthornw** [comment]
Agreed that a dedicated endpoint (and button in the UI) would be good. 
The dataframe format is supported in the SDK (but it's really just a composition of endpoints rather than a dedicated endpoint) - that PR adds the abiltiy to export as json/other formats.

Semi-related discussion:
https://github.com/langchain-ai/langsmith-sdk/issues/537
